### ISSUE TITLE
Add CSS linting

### ISSUE THREAD

**Issue by meiamsome** [CodingTrain/project-trainsite-frontend#16]
I think we can do this with [stylelint-processor-styled-components](https://github.com/styled-components/stylelint-processor-styled-components), but I have never set it up

**Comment by Fraasi** [comment]
I got the basic setup set up (:smile:). Only got a few font-family errors:

```
$ npm run lint:css

> project-trainsite-frontend@0.1.0 lint:css G:\Code\GitClones\project-trainsite-frontend
> stylelint src/**/*.tsx


src/components/SupportTheCodingTrain/index.styles.tsx
 4:16  ×  Unexpected missing      font-family-no-missing-generic-family-keyword
          generic font family

npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! project-trainsite-frontend@0.1.0 lint:css: `stylelint src/**/*.tsx`
npm ERR! Exit status 2
npm ERR!
npm ERR! Failed at the project-trainsite-frontend@0.1.0 lint:css script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\Fra.AS.I\AppData\Roaming\npm-cache\_logs\2019-01-25T18_44_34_232Z-debug.log
```

Easy enough to fix just adding sans-serif like this `font-family: cubanoregular, sans-serif;`  
But it seems stylelint errors pretty verbose with all those 'npm err's, we can turn it off rule by rule or use `"defaultSeverity": "warning"` -setting to mute those omg my computer exploded -errors.  

So the question is, what config do we want to use?  
I went with the [recommended](https://github.com/stylelint/stylelint-config-recommended) as it was in the front page setup info. But there is also the [standard](https://github.com/stylelint/stylelint-config-standard) which might be better.

Oh, there's also [a11y-plugin for stylelint](https://github.com/YozhikM/stylelint-a11y).

**Comment by meiamsome** [comment]
I think go for standard for now, so that we're a bit more strict than recommended but not too strict. 

Ignore the `npm ERR!` messages, that's just npm being very verbose, but it is important for the CI testing that those are actually errors.

Feel free to open a PR

**Comment by meiamsome** [comment]
Fixed in #22
